src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$cell-size: 100px;
+$coll-couny: 7;
+
+$date: mon 0,
```

Comment:
The 'mon' value should not be 0. It should follow the same pattern as other days, i.e., 'mon $cell-size + 0'.